satpy/readers/clavrx.py

```diff
@@ -235,14 +268,24 @@ def _read_axi_fixed_grid(filename: str, l1b_attr) -> geometry.AreaDefinition:
         x, y = l1b["x"], l1b["y"]
         area_extent, ncols, nlines = _CLAVRxHelper._area_extent(x, y, h)
 
-        area = geometry.AreaDefinition(
-            "ahi_geos",
-            "AHI L2 file area",
-            "ahi_geos",
-            proj,
-            ncols,
-            nlines,
-            np.asarray(area_extent))
+        if sensor == "abi":
+            area = geometry.AreaDefinition(
+              "abi_geos",
+              "ABI L2 file area",
+              "abi_geos",
+              proj,
+              ncols,
+              nlines,
+              np.asarray(area_extent))
```

Comment:
Not sure why this `np.asarray` was ever a thing, but area_extent should always be a tuple when passed to AreaDefinition.